### PR TITLE
fix: fix page fault for TDinsight v3 in grafana

### PR DESCRIPTION
**Pull request by sunpe**

### CODE REVIEW

File: dashboards/TDinsightV3.json [hunk 1]
```diff
@@ -114,27 +126,29 @@
...
+        "code": {
+          "language": "plaintext",
+          "showLineNumbers": false,
+          "showMiniMap": false
+        },
         "content": "<h1><center>TDengine Cluster Dashboard (First EP: ${firstEp}, Version: ${version})</center></h1>",
         "mode": "markdown"
       },
-      "pluginVersion": "7.5.10",
+      "pluginVersion": "9.2.2",
```
Comment (sangshuduo) [inline dashboards/TDinsightV3.json:137]: Not sure we should use the grafana version here. @zitsen, please comment

File: dashboards/TDinsightV3.json [hunk 2]
```diff
@@ -198,11 +212,12 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "7.5.10",
+      "pluginVersion": "9.2.2",
```
Comment (sangshuduo) [inline dashboards/TDinsightV3.json:215]: same here

File: dashboards/TDinsightV3.json [hunk 3]
```diff
@@ -257,11 +272,12 @@
...
```
Comment (sangshuduo) [inline dashboards/TDinsightV3.json:275]: here

File: dashboards/TDinsightV3.json [hunk 4]
```diff
@@ -389,11 +406,12 @@
...
```
Comment (sangshuduo) [inline dashboards/TDinsightV3.json:409]: here

File: dashboards/TDinsightV3.json [hunk 5]
```diff
@@ -450,6 +468,8 @@
       "id": 100,
       "options": {
         "displayMode": "gradient",
+        "minVizHeight": 10,
+        "minVizWidth": 0,
```
Comment (sangshuduo) [inline dashboards/TDinsightV3.json:472]: why min height is more than 0 but min width can be 0?

**Review by sangshuduo — COMMENTED** [review]
see my comments

File: dashboards/TDinsightV3.json [hunk 6]
```diff
@@ -5026,21 +4767,20 @@
...
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
-        "allValue": null,
-        "current": {},
+        "current": {
+          "selected": false,
+          "text": "934293411e15:6030",
```
Comment (huolibo) [inline dashboards/TDinsightV3.json:4777]: this value  is generic?

File: dashboards/TDinsightV3.json [hunk 7]
```diff
@@ -5002,21 +4744,20 @@
...
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
-        "allValue": null,
-        "current": {},
+        "current": {
+          "selected": false,
+          "text": "3.0.1.5",
```
Comment (huolibo) [inline dashboards/TDinsightV3.json:4754]: define a variable for version.

**Review by huolibo — COMMENTED** [review]
see my comment

**Review by huolibo — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**